version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.1.7-SNAPSHOT"
\ No newline at end of file
+version in ThisBuild := "0.2.0"
```

Comment:
ok, what do you propose as version number? 

btw, is the release process similar to what we do @Wehkamp?